addons/stock_account/tests/test_stockvaluationlayer.py

```diff
@@ -596,6 +595,27 @@ def test_dropship_1(self):
         self.assertEqual(self.product1.quantity_svl, 2)
         self.assertEqual(orig_standard_price, self.product1.standard_price)
 
+    def test_standard_price_fifo_1(self):
+        self._make_in_move(self.product1, 3, unit_cost=17)
+        self._make_in_move(self.product1, 1, unit_cost=23)
+        self._make_out_move(self.product1, 3, create_picking=True)
+        self.assertEqual(self.product1.standard_price, 23)
+
+    def test_standard_price_fifo_2(self):
+        self._make_in_move(self.product1, 5, unit_cost=17)
+        self._make_in_move(self.product1, 1, unit_cost=23)
+        self._make_out_move(self.product1, 4, create_picking=True)
+        self.assertEqual(self.product1.standard_price, 17)
```

Comment:
can you move these two tests to test_stockvaluation.py?
this file is reserved to the implementation of our valuation spreadsheet